src/js/controllers/auth/securityQuestionController.js

```diff
@@ -216,8 +217,9 @@
 
             $scope.initNavigator.on('postpush', () => {
                 $timeout(() => {
-                    const securityAnswer = document.getElementById('security_answer_input');
-                    if (securityAnswer) {
+                    const securityAnswer = document.getElementById('security-answer-input');
+                    // iOS blocks focus events and causes issues focussing the input field afterwards
+                    if (securityAnswer && ons.platform.isIOS()) {
                         securityAnswer.focus();
                     }
```

Comment:
> The focus() call causes the input field to not be directly selectable on iOS.

If that's true, shouldn't this be `!ons.platform.isIOS()`? To avoid focusing on iOS?

There's another issue though, because of how onsen is designed, we get `ons.platform.isIOS() === true` on browser. That's why in many places, we have to do a double check: `Constants.app && ons.platform.isIOS()`, to detect a mobile app running iOS.

Reply:
I was looking at the changes and noticed too that I accidentally inverted the logic. Interestingly, it still worked fine when I tested it. So now I reverted back to the original condition.

Reply:
Ok, my bad. Because the ID didn't exist the problem was "fixed". It's now actually fixed.

> There's another issue though, because of how onsen is designed, we get ons.platform.isIOS() === true on browser. That's why in many places, we have to do a double check: Constants.app && ons.platform.isIOS(), to detect a mobile app running iOS.

I tested this and it works fine without the additional condition.

Reply:
It might work fine on iOS, but when running on browser, it no longer auto-focuses now, which was a nice feature.

Could you import Constants and add the condition, just to preserve the browser functionality?

Reply:
It does auto-focus for me in the browser. How can I reproduce it?

Unless you mean the browser **on** a mobile device!?

Reply:
No, I just mean a regular browser on a computer. For me on Windows, it doesn't auto-focus. I'll send you a direct message.

Reply:
Added the condition. Might be a Chrome on Windows specific issue with Onsen v1.

Reply:
Thanks!